ihmc-perception/src/main/java/us/ihmc/perception/gpuHeightMap/RapidHeightMapExtractor.java

```diff
@@ -587,6 +587,11 @@ public void createSteppabilityMapImages(int height, int width)
       steppabilityConnectionsImage.createOpenCLImage(openCLManager, OpenCL.CL_MEM_READ_WRITE);
    }
 
+   public void destroy()
+   {
+      // TODO What do we need to destroy in here?
```

Comment:
Yeah this thing never destroyed anything, we need to watch that

Reply:
Did you add an empty destroy() method?

Reply:
Yep, needs to be filled in sooner than later

Reply:
Probably should fill this out before merging, right?

Reply:
Can you guys help me find stuff to destroy? I'm not sure what would need to be destroyed